sky/utils/common_utils.py

```diff
@@ -291,3 +291,15 @@ def decode_payload(payload_str: str) -> Any:
     payload_str = matched[0]
     payload = json.loads(payload_str)
     return payload
+
+
+def class_fullname(cls):
+    """Get the full name of a class.
```

Comment:
Nice! The class printing has been ongoing for a while.

nit: Maybe useful to include an example input/output value.

Reply:
Added. Thanks!